pyproject.toml

```
@@ -23,11 +23,12 @@ readme = "README.rst"
 requires-python = ">=3.7"
 
 [project.optional-dependencies]
-all = ["getdaft[aws, azure, gcp, ray, pandas, numpy, iceberg, deltalake, sql]"]
+all = ["getdaft[aws, azure, gcp, ray, pandas, numpy, hudi, iceberg, deltalake, sql]"]
```

Oh let's not include `hudi` in `all` for now because it will mean that the pyarrow version gets pinned in `all` :(

```suggestion
all = ["getdaft[aws, azure, gcp, ray, pandas, numpy, iceberg, deltalake, sql]"]
```

This should be fine because not many people use `all`